api/src/main/resources/workbench-api.yaml

```diff
@@ -6787,6 +6814,38 @@ definitions:
           which means it is a relative path in the Jupyter user-facing file system,
           e.g. "foo/bar.ipynb".'
 
+  AppLocalizeRequest:
+    type: object
+    required:
+    - notebookNames
+    - playgroundMode
+    properties:
+      fileNames:
+        type: array
+        description: 'Names of notebooks to localize. This is just the basename (no
+          gs:// needed). This is interpreted as relative to the /notebooks directory
+          within the provided workspace''s Google Cloud Storage bucket.'
+        items:
+          type: string
+      playgroundMode:
+        type: boolean
+        default: false
+        description: Set to true if playgroundMode needed
+      appType:
```

Comment:
Do we need appType?

Reply:
AppType is not needed now, but would be needed once we support SAS. Adding appType here might be easier once we support more apps